INSTALL.md

```diff
@@ -0,0 +1,89 @@
+# Installing device plugins to cluster
+
+## Install device plugins via a daemonset
+
+Each plugin can be installed via a daemonset. The install changes slightly based on the desired plugin. See install instructions per [plugin](README.md#plugins).
+
+Installing plugins via daemonsets deployes them to the ```default``` (or currently active) namespace. Use kubectl's ```--namespace``` argument to change the deployment namespace.
+
+## Install device plugins via device plugin operator
+
+A more advanced install method is via device plugin operator. There the plugins are installed by supplying device plugin CRDs which the operator then converts into plugin deployments. See installation instructions in the [operator README](cmd/operator/README.md#installation).
+
+Device plugins installed via device plugin operator are installed in the device plugin namespace ```inteldeviceplugins-system```.
```

Comment:
More simply:
> Operator installs device plugins to the ```inteldeviceplugins-system``` namespace.

Btw. Is this overridable by CRD?   If yes, add "... by default".

Reply:
CRD doesn't have a field for namespace, so no?

Reply:
Yep, it's up to the chart to handle namespaces. For example, prometheus can be installed as a dependency into a different namespace. But you'll have to supply the ns via values, not via helm install arguments.